langchain/prompts/base.py

```diff
@@ -12,6 +12,8 @@
 from langchain.formatting import formatter
 from langchain.schema import BaseMessage, BaseOutputParser, HumanMessage, PromptValue
 
+ACCEPTABLE_PARTIAL_TYPES = Union[str, Callable[[], str], "BasePromptTemplate"]
```

Comment:
annotations is imported from the future

```suggestion
ACCEPTABLE_PARTIAL_TYPES = Union[str, Callable[[], str], BasePromptTemplate]
```